lib/litani_report.py

```diff
@@ -410,6 +411,35 @@ def __call__(self, run):
 
 
 
+async def release_html_dir(args):
+    lockable_dir = lib.litani.LockableDirectory(args.dir)
+    lockable_dir.release()
+
+
+async def acquire_html_dir(args):
+    remaining = args.timeout
+    while True:
+        html_dir = lib.litani.get_report_dir().resolve()
+        lockable_dir = lib.litani.LockableDirectory(html_dir)
+        try:
+            if lockable_dir.acquire():
```

Comment:
Going on a slight tangent.

I was looking at `lib/litani.py`:

```python
class LockableDirectory:
    """POSIX-compliant directory locking"""

# ...

    def acquire(self):
        """Return True if acquisition was successful, False otherwise"""
        try:
            self._lock_file.unlink()
        except OSError:
            return False
        else:
            return True
```

Why does `acquire()` handle only the case of `OSError` and not `FileNotFoundError` ?

Reply:
`OSError` is a superclass of `FileNotFoundError`, which means that the catch here is redundant. Thanks.